tests/benchmark/.gitignore

```diff
@@ -0,0 +1,3 @@
+avro/*.prof
+avro/tmp-KafkaCluster
```

Comment:
It'd be better to use a common TRIVUP_ROOT for all tests, otherwise kafka 2.4.0 (or wahtever) is downloaded into each of these tmp dirs.